.travis.yml

```diff
@@ -83,13 +83,15 @@ script: # All commands must exit with code 0 on success. Anything else is consid
   - if [ $BUILDER == catkin ]; then export TARGET_PKG=`find build*/*hiro* -name Makefile -print |  sed s@.*/\\\\\([^\/]*\\\\\)/Makefile@\\\1@g` ; fi
   # - if [ $BUILDER == catkin ]; then source devel/setup.sh; export EXIT_STATUS=0; for pkg in $TARGET_PKG; do (roscd $pkg; [ "`find . -iname '*.test'`" == "" ] && echo "[$pkg] No tests were found!!!"  || find . -iname "*.test" -print0 | xargs -0 -n1 rostest || export EXIT_STATUS=$?;) done; [ $EXIT_STATUS == 0 ] ; fi
   - export EXIT_STATUS=$?
+  - if [ $BUILDER == catkin ]; then source devel*/setup.bash              ; fi
```

Comment:
Assuming the asterisk (devel`*`) is to cover both `devel` and `devel_isolated` spaces.